src/clj/web/stats.clj

```diff
@@ -0,0 +1,102 @@
+(ns web.stats
+  (:require [web.db :refer [db object-id]]
+            [monger.collection :as mc]
+            [web.utils :refer [response]])
+  (:import org.bson.types.ObjectId))
+
+(defn clear-user-stats
+  "Clear any statistics for a given user-id contained in a request"
+  [req]
```

Comment:
Ideally only routes that directly handle an HTTP request should take a request (`req`) as a parameter. This function should take a user id only and operate directly on that.